### PR TITLE
Suggested edits for Tiny-4398

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: _includes/configuration/image_toolbar.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+{{ site.requires_5_2v }}
+
+The **quickbars_image_toolbar** option configures the Quick Image toolbar provided by the [quickbars plugin]({{ site.baseurl }}/plugins/quickbars). To change the buttons on the Quick Image toolbar, provide a space-separated string of [toolbar button names]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols). The Quick Image toolbar is intended for image-related buttons, but any [{{site.productname}} toolbar buttons]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols) or [custom toolbar buttons]({{ site.baseurl }}/ui-components/toolbarbuttons) are allowed.
```
Comment (metricjs) [inline _includes/configuration/image_toolbar.md]: You lost the "or disable by setting it to false" point?

File: _includes/configuration/insert-toolbar.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
-## quickbars_insert_toolbar
+### quickbars_insert_toolbar
 
-The **quickbars_insert_toolbar** option enables specifying toolbar items to include on the Quick Insert toolbar enabled by the [quickbars]({{ site.baseurl }}/plugins/quickbars) plugin. It is recommended to only have controls in this toolbar related to inserting content. However, nothing is restricting to use any of the available [toolbar controls]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols).
+The **quickbars_insert_toolbar** option configures the Quick Insert toolbar provided by the [quickbars plugin]({{ site.baseurl }}/plugins/quickbars). To change the buttons on the Quick Insert toolbar, provide a space-separated string of [toolbar button names]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols). The Quick Insert toolbar is intended for buttons related to inserting content, but any [{{site.productname}} toolbar buttons]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols) or [custom toolbar buttons]({{ site.baseurl }}/ui-components/toolbarbuttons) are allowed.
```
Comment (metricjs) [inline _includes/configuration/insert-toolbar.md]: Also disable by false

File: _includes/configuration/selection-toolbar.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
-## quickbars_selection_toolbar
+### quickbars_selection_toolbar
 
-This option enables specifying toolbar items to include on the Quick Selection toolbar or to disable the toolbar. 
-
-It is recommended that only formatting related controls are specified in this toolbar. However, any of the available [toolbar controls]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols) can also be used.
+The **quickbars_selection_toolbar** option configures the Quick Selection toolbar provided by the [quickbars plugin]({{ site.baseurl }}/plugins/quickbars). To change the buttons on the Quick Selection toolbar, provide a space-separated string of [toolbar button names]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols). The Quick Selection toolbar is intended for buttons related to formatting content, but any [{{site.productname}} toolbar buttons]({{ site.baseurl }}/advanced/editor-control-identifiers/#toolbarcontrols) or [custom toolbar buttons]({{ site.baseurl }}/ui-components/toolbarbuttons) are allowed.
```
Comment (metricjs) [inline _includes/configuration/selection-toolbar.md]: Also disable with false.

File: _includes/configuration/image_toolbar.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/configuration/image_toolbar.md]: I moved it down to the just before the codeblock

File: _includes/configuration/image_toolbar.md [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
 **Defaults:** `alignleft aligncenter alignright`
 
-##### Example customizing the Quick Image toolbar
+#### Example customizing the Quick Image toolbar
 
 ```js
 tinymce.init({
   selector: 'div.tinymce',
-  plugins: 'quickbars',
+  plugins: 'quickbars imagetools',
```
Comment (lnewson) [inline _includes/configuration/image_toolbar.md]: Maybe add the `image` plugin here as well. I don't think it's needed, but if you're using imagetools you'd probably want to be using image as well.

Comment (tylerkelly13) [reply _includes/configuration/image_toolbar.md]: image has been added to the plugins string

**Pull request merged**